insights/specs/datasources/luks_devices.py

```diff
@@ -123,3 +124,28 @@ def luks_data_sources(broker):
         return datasources
 
     raise SkipComponent
+
+
+@datasource(LuksDump, HostContext)
```

Comment:
@xiangce @danzatt why not just use parsers and a collector instead of adding a datasource that has to run on a customer systems?

Reply:
@danzatt, yeap, what @bfahr suggested makes sense to me. It seems possible to run the `luksmeta -d` command against all the `/dev/disk/by-uuid/*` devices. And then get what you wanted (uuids where ["header"]["Version"] == "1" in LuksDump) via a combiner.